agcaballero/portfolio/src/main/webapp/script.js

```diff
@@ -12,19 +12,27 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/* Returns the link to a random post on the blog */
+/** Returns an array of post links to be used in randomPost() */
+function postLinks() {
+  return ['#post1-lnk', '#post2-lnk', '#post3-lnk', '#post4-lnk'];
+}
+
+/** Returns the link to a random post on the blog */
 function randomPost() {
   // array with all post links
-  const posts = ['#post1-lnk', '#post2-lnk', '#post3-lnk', '#post4-lnk'];
+  const posts = postLinks();
 
   // returns a random link
   return posts[Math.floor(Math.random() * posts.length)];
 }
 
-/* Tests randomPost() by verifying it spits out only one of the random post
-options & logging the percentage of times it returns each */
+/**
+Tests randomPost() by verifying it spits out only one of the random post
+options & logging the percentage of times it returns each
+*/
 function testRandomPost(trials = 10000) {
-  const LENGTH = 4;
+  const postLinks = postLinks();
+  const postLinkCount = postLinks.length;
   const sums = [0, 0, 0, 0];
```

Comment:
Is there a way to make the number of elements in `sums` depend on `postLinkCount` too? For example, if another item was added to `postLinks`, this would need to have 5 elements.

Reply:
I think this would just be a declaration plus a simple for loop - I'll change it.

Reply:
Done